packages/react-scripts/layout/views/layout.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="<%= simpleLocale %>">
+<html dir="<%= typeof languageDir !== 'undefined' ? languageDir : 'ltr' %>" lang="<%= simpleLocale %>">
```

Comment:
@kyle-west would it be better to always have dir, and have it be ltr, or do we just add `dir="rtl"` when it's not the default and omit dir for ltr?